notebooks/run_predictions.ipynb

```diff
@@ -0,0 +1 @@
h #350, batch_size 16, seq length 1774]\tLoss: 0.031458\n","374it [00:14, 21.19it/s]Train epoch: 144 [batch #375, batch_size 16, seq length 1893]\tLoss: 0.032233\n","398it [00:16, 20.50it/s]Train epoch: 144 [batch #400, batch_size 16, seq length 2032]\tLoss: 0.031286\n","424it [00:17, 19.64it/s]Train epoch: 144 [batch #425, batch_size 16, seq length 2231]\tLoss: 0.028736\n","450it [00:18, 18.57it/s]Train epoch: 144 [batch #450, batch_size 16, seq length 2491]\tLoss: 0.031891\n","474it [00:20, 17.53it/s]Train epoch: 144 [batch #475, batch_size 16, seq length 2500]\tLoss: 0.036871\n","500it [00:21, 15.71it/s]Train epoch: 144 [batch #500, batch_size 16, seq length 2500]\tLoss: 0.034849\n","505it [00:22, 22.79it/s]\n","epoch loss: 0.029604872466424608\n","file for evaluation: ../../mimicdata/mimic3/dev_50.csv\n","1573it [00:04, 350.85it/s]\n","\n","[MACRO] accuracy, precision, recall, f-measure, AUC\n","0.3734, 0.5646, 0.4839, 0.5211, 0.8633\n","[MICRO] accuracy, precision, recall, f-measure, AUC\n","0.4033, 0.6306, 0.5281, 0.5748, 0.8849\n","rec_at_5: 0.5499\n","prec_at_5: 0.5627\n","\n","saved metrics, params, model to directory /content/gdrive/Othercomputers/Pebble-MBP16/mcs/cs598-healthcare/paper-project/code/CS598DL4H/caml-mimic/mimicdata/saved_models/conv_attn_May_03_021908\n","\n","EPOCH 145\n","0it [00:00, ?it/s]/content/gdrive/Othercomputers/Pebble-MBP16/mcs/cs598-healthcare/paper-project/code/CS598DL4H/caml-mimic/datasets.py:82: VisibleDeprecationWarning: Creating an ndarray from ragged nested sequences (which is a list-or-tuple of lists-or-tuples-or ndarrays with different lengths or shapes) is deprecated. If you meant to do this, you must specify 'dtype=object' when creating the ndarray.\n","  np.array(self.descs)\n","Train epoch: 145 [batch #0, batch_size 16, seq length 212]\tLoss: 0.069467\n","23it [00:00, 30.38it/s]Train epoch: 145 [batch #25, batch_size 16, seq length 571]\tLoss: 0.023478\n","48it [00:01, 28.52it/s]Train epoch: 145 [batch #50, batch_size 16, seq length 709]\tLoss: 0.021578\n","74it [00:02, 28.65it/s]Train epoch: 145 [batch #75, batch_size 16, seq length 806]\tLoss: 0.032496\n","100it [00:03, 28.04it/s]Train epoch: 145 [batch #100, batch_size 16, seq length 892]\tLoss: 0.026319\n","124it [00:04, 26.56it/s]Train epoch: 145 [batch #125, batch_size 16, seq length 978]\tLoss: 0.025248\n","148it [00:05, 26.38it/s]Train epoch: 145 [batch #150, batch_size 16, seq length 1063]\tLoss: 0.029083\n","175it [00:06, 25.30it/s]Train epoch: 145 [batch #175, batch_size 16, seq length 1137]\tLoss: 0.029453\n","199it [00:07, 25.16it/s]Train epoch: 145 [batch #200, batch_size 16, seq length 1217]\tLoss: 0.028557\n","223it [00:08, 24.10it/s]Train epoch: 145 [batch #225, batch_size 16, seq length 1299]\tLoss: 0.026255\n","250it [00:09, 24.23it/s]Train epoch: 145 [batch #250, batch_size 16, seq length 1383]\tLoss: 0.027134\n","274it [00:10, 23.04it/s]Train epoch: 145 [batch #275, batch_size 16, seq length 1470]\tLoss: 0.030652\n","298it [00:
```

Comment:
Can you try to format this? I think the `json_pp` CLI tool would do it.

Reply:
I think this was output?  If this is still gross let me know it looked ok but I wasn't entirely sure what I was looking for.